salt/preflight/map.jinja

```diff
@@ -0,0 +1,5 @@
+{% import_yaml 'preflight/defaults.yaml' as settings with context %}
+
+{% set defaults = salt['grains.filter_by']({
+    'settings': settings
```

Comment:
Here you need to put `'default'` as key instead of settings.

Maybe make sense to make this part of the global map.jinja (IMO depends if you want to have a dedicated defaults.yaml for preflight or no)